php_mysqlnd_azure.h

```diff
@@ -28,14 +28,22 @@
 extern zend_module_entry mysqlnd_azure_module_entry;
 #define phpext_mysqlnd_azure_ptr &mysqlnd_azure_module_entry
 
-#define EXT_MYSQLND_AZURE_NAME      "mysqlnd_azure"
-#define EXT_MYSQLND_AZURE_VERSION   "1.0.3"
+#define PHP_MYSQLND_AZURE_NAME      "mysqlnd_azure"
+#define PHP_MYSQLND_AZURE_VERSION   "1.1.0RC1"
+
+#define STRING_EQUALS(z_str,str) (ZSTR_LEN((z_str)) == strlen((str)) && strcasecmp((str), ZSTR_VAL((z_str))) == 0)
 
 ZEND_BEGIN_MODULE_GLOBALS(mysqlnd_azure)
-    zend_bool       enabled;
+    zend_bool       enableRedirect;
```

Comment:
Thanks for checking. This is by mistake. I have changed the value type to enum mysqlnd_azure_redirect_mode. The code still worked somehow since zend_bool and the way to parse option works differently...